### PR TITLE
Fix font weight, sentence case, etc to align with v5 theme

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

This PR:
- Fixes "Actions" button example
- Removes anchor "weight" example
- Changes use of weight="bold" to TextEmphasis
- Sentence cases things that were title case

#### Where should the reviewer start?
Anywhere.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### What are the relevant issues?
Related to #3274

### CODE REVIEW

File: aries-site/src/examples/templates/code-blocks/CodeBlocks.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const Identifier = ({ title, subtitle, icon }) => (
   >
     <Box pad={{ vertical: 'xsmall' }}>{icon}</Box>
     <Box>
-      <Text color="text-strong" size="xxlarge" weight="bold">
+      <Text color="text-strong" size="xxlarge" weight={500}>
```
Comment (halocline) [inline aries-site/src/examples/templates/code-blocks/CodeBlocks.js:63]: TextEmphasis size="xxlarge" instead?

File: aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js [hunk 2]
```diff
@@ -85,9 +86,9 @@ const Orders = () => {
       >
         {(datum, index) => (
           <Box gap="xsmall" justify="between" key={index}>
-            <Text color="text-strong" size="large" weight="bold">
+            <TextEmphasis color="text-strong" size="large">
```
Comment (halocline) [inline aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js:89]:
```suggestion
            <TextEmphasis size="large">
```

File: aries-site/src/examples/templates/global-header/components/HeaderNav.js [hunk 3]
```diff
@@ -60,9 +61,9 @@ const UserDetails = () => {
           <Avatar src={user.image} size="large" />
         )}
         <Box pad={{ vertical: 'small' }}>
-          <Text color="text-strong" size="large" weight="bold">
+          <TextEmphasis size="large" color="text-strong">
```
Comment (halocline) [inline aries-site/src/examples/templates/global-header/components/HeaderNav.js:64]:
```suggestion
          <TextEmphasis size="large">
```

File: aries-site/src/examples/templates/global-header/components/MenuLayer.js [hunk 4]
```diff
@@ -115,9 +114,9 @@ const SidebarFooter = () => (
             <Aruba color="background" />
           </Box>
           <Box>
-            <Text weight="bold" color="text-strong" size="xsmall">
+            <TextEmphasis color="text-strong" size="xsmall">
```
Comment (halocline) [inline aries-site/src/examples/templates/global-header/components/MenuLayer.js:117]:
```suggestion
            <TextEmphasis size="xsmall">
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 5]
```diff
@@ -14,9 +15,7 @@ export const HeaderFooterExample = () => (
           // remove dashed border in prod, for demo purposes only
           border={{ color: 'border', style: 'dashed' }}
         >
-          <Text weight="bold" color="text-strong">
-            Header
-          </Text>
+          <TextEmphasis color="text-strong">Header</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:18]:
```suggestion
          <TextEmphasis>Header</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 6]
```diff
@@ -27,9 +26,7 @@ export const HeaderFooterExample = () => (
           // remove dashed border in prod, for demo purposes only
           border={{ color: 'border', style: 'dashed' }}
         >
-          <Text weight="bold" color="text-strong">
-            Main
-          </Text>
+          <TextEmphasis color="text-strong">Main</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:29]:
```suggestion
          <TextEmphasis>Main</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 7]
```diff
@@ -40,9 +37,7 @@ export const HeaderFooterExample = () => (
           // remove height="xxsmall" in prod, for demo purposes only
           height="xxsmall"
         >
-          <Text weight="bold" color="text-strong">
-            Footer
-          </Text>
+          <TextEmphasis color="text-strong">Footer</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:40]:
```suggestion
          <TextEmphasis>Footer</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 8]
```diff
@@ -14,9 +15,7 @@ export const HeaderOnlyExample = () => (
           // remove height="xxsmall" in prod, for demo purposes only
           height="xxsmall"
         >
-          <Text weight="bold" color="text-strong">
-            Header
-          </Text>
+          <TextEmphasis color="text-strong">Header</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:18]:
```suggestion
          <TextEmphasis>Header</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 9]
```diff
@@ -27,9 +26,7 @@ export const HeaderOnlyExample = () => (
           // remove height="xlarge" in prod, for demo purposes only
           height="xlarge"
         >
-          <Text weight="bold" color="text-strong">
-            Main
-          </Text>
+          <TextEmphasis color="text-strong">Main</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:29]:
```suggestion
          <TextEmphasis>Main</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 10]
```diff
@@ -13,9 +14,7 @@ export const StickyHeaderExample = () => (
         // remove height="xxsmall" in prod, for demo purposes only
         height="xxsmall"
       >
-        <Text weight="bold" color="text-strong">
-          Header
-        </Text>
+        <TextEmphasis color="text-strong">Header</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:17]:
```suggestion
        <TextEmphasis>Header</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 11]
```diff
@@ -27,9 +26,7 @@ export const StickyHeaderExample = () => (
           // remove height="xlarge" in prod, for demo purposes only
           height="xlarge"
         >
-          <Text weight="bold" color="text-strong">
-            Main
-          </Text>
+          <TextEmphasis color="text-strong">Main</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:29]:
```suggestion
          <TextEmphasis>Main</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 12]
```diff
@@ -40,9 +37,7 @@ export const StickyHeaderExample = () => (
           // remove height="xxmsmall" in prod, for demo purposes only
           height="xxsmall"
         >
-          <Text weight="bold" color="text-strong">
-            Footer
-          </Text>
+          <TextEmphasis color="text-strong">Footer</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:40]:
```suggestion
          <TextEmphasis>Footer</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/anatomy/PageContainerFull.js [hunk 13]
```diff
@@ -44,9 +45,7 @@ export const PageContainerFull = () => {
         pad={{ horizontal: 'xsmall' }}
         background="orange"
       >
-        <Text color="text-strong" weight="bold">
-          {PAGE_CONTAINER_WIDTH}
-        </Text>
+        <TextEmphasis color="text-strong">{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/PageContainerFull.js:48]:
```suggestion
        <TextEmphasis>{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/anatomy/PageContainerNarrow.js [hunk 14]
```diff
@@ -44,9 +45,7 @@ export const PageContainerNarrow = () => {
         pad={{ horizontal: 'xsmall' }}
         background="orange"
       >
-        <Text color="text-strong" weight="bold">
-          {PAGE_CONTAINER_WIDTH}
-        </Text>
+        <TextEmphasis color="text-strong">{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/PageContainerNarrow.js:48]:
```suggestion
        <TextEmphasis>{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/anatomy/PageContainerWide.js [hunk 15]
```diff
@@ -40,9 +41,7 @@ export const PageContainerWide = () => {
     >
       <LinkPrevious id="left-edge" color="text-strong" />
       <Box id="label" pad={{ horizontal: 'xsmall' }} background="orange">
-        <Text color="text-strong" weight="bold">
-          {PAGE_CONTAINER_WIDTH}
-        </Text>
+        <TextEmphasis color="text-strong">{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/PageContainerWide.js:44]:
```suggestion
        <TextEmphasis>{PAGE_CONTAINER_WIDTH}</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js [hunk 16]
```diff
@@ -38,9 +39,7 @@ const GridViewPort = ({ columns, label, marginLabel, rows, width }) => (
         background="status-unknown"
         border={{ style: 'dashed' }}
       >
-        <Text color="text-strong" weight="bold">
-          {label}
-        </Text>
+        <TextEmphasis color="text-strong">{label}</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js:42]:
```suggestion
        <TextEmphasis>{label}</TextEmphasis>
```

File: aries-site/src/examples/templates/page-layouts/anatomy/components/ContentArea.js [hunk 17]
```diff
@@ -30,14 +31,14 @@ export const ContentArea = ({
       {icon ? (
         <Box align="center" gap="small" direction="row">
           <Hpe color="brand" />
-          <Text color="text-strong" size="small" weight="bold">
+          <TextEmphasis color="text-strong" size="small">
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/components/ContentArea.js:34]:
```suggestion
          <TextEmphasis size="small">
```

File: aries-site/src/examples/templates/page-layouts/anatomy/components/ContentArea.js [hunk 18]
```diff
@@ -30,14 +31,14 @@ export const ContentArea = ({
...
             {title}
-          </Text>
+          </TextEmphasis>
         </Box>
       ) : (
-        <Text color="text-strong" size="small" weight="bold">
+        <TextEmphasis color="text-strong" size="small">
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/anatomy/components/ContentArea.js:39]:
```suggestion
        <TextEmphasis size="small">
```

File: aries-site/src/layouts/content/Example/DoDontContainer.js [hunk 19]
```diff
@@ -31,9 +32,9 @@ export const DoDontContainer = ({
           round="xxsmall"
         />
         <Box>
-          <Text color="text-strong" weight="bold" size="large">
+          <TextEmphasis color="text-strong" size="large">
```
Comment (halocline) [inline aries-site/src/layouts/content/Example/DoDontContainer.js:35]:
```suggestion
          <TextEmphasis size="large">
```

File: aries-site/src/layouts/content/StyleTable.js [hunk 20]
```diff
@@ -39,9 +40,7 @@ export const StyleTable = ({ data }) => {
                 scope="col"
                 pad={{ vertical: 'small' }}
               >
-                <Text color="text-strong" weight="bold">
-                  {c.render}
-                </Text>
+                <TextEmphasis color="text-strong">{c.render}</TextEmphasis>
```
Comment (halocline) [inline aries-site/src/layouts/content/StyleTable.js:43]:
```suggestion
                <TextEmphasis>{c.render}</TextEmphasis>
```

File: aries-site/src/layouts/navigation/SearchResults.js [hunk 21]
```diff
@@ -126,7 +126,7 @@ export const SearchResults = ({
               <Text
```
Comment (halocline) [inline aries-site/src/layouts/navigation/SearchResults.js:126]: TextEmphasis?

File: aries-site/src/components/content/MarkdownComponents.js [hunk 22]
```diff
@@ -123,9 +124,7 @@ export const components = {
       pad={{ horizontal: 'small', vertical: 'xsmall' }}
       {...props}
     >
-      <Text color="text-strong" weight="bold">
-        {props.children}
-      </Text>
+      <TextEmphasis color="text-strong">{props.children}</TextEmphasis>
```
[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/components/content/MarkdownComponents.js:127]: After discussion, it seems the 80%+ case is that text-strong would be used with weight 500 for emphasis. Made the appropriate adjustments.

File: aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js [hunk 23]
```diff
@@ -66,7 +67,7 @@ export const NameValueListEditHorizontalExample = () => {
       <Wrapper {...wrapperProps}>
         <NameValueList valueProps={{ width: ['auto', 'medium'] }}>
           {Object.entries(currentData).map(([name, value]) => {
-            let nameRender = <Text weight="bold">{value.displayName}</Text>;
+            let nameRender = <TextEmphasis>{value.displayName}</TextEmphasis>;
```
[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js:70]: Correct, I will remove TextEmpahsis.

File: aries-site/src/examples/components/page/PageExample.js [hunk 24]
```diff
@@ -24,19 +24,19 @@ export const PageExample = () => (
     >
       <ContentArea title="Page header" border background="purple!" />
       <ContentArea title="Other content" border background="orange">
-        <Paragraph margin={{ bottom: 'none' }}>
+        <Paragraph color="text-strong" margin={{ bottom: 'none' }}>
           Page is a container providing context (i.e. instructions) for how
           content should be laid out on a page.
         </Paragraph>
-        <Paragraph margin={{ bottom: 'none' }}>
+        <Paragraph color="text-strong" margin={{ bottom: 'none' }}>
```
[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/examples/components/page/PageExample.js:31]: Yes, exactly.

File: aries-site/src/layouts/navigation/SearchResults.js (hunk 21 above)
Comment (taysea) [reply aries-site/src/layouts/navigation/SearchResults.js]: Done.

File: aries-site/src/examples/templates/code-blocks/CodeBlocks.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/examples/templates/code-blocks/CodeBlocks.js]: Done.

File: aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js (hunk 23 above)
[the remark this answers is not in the document]
Comment (taysea) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListEditHorizontalExample.js]: Done.

**Review by halocline — APPROVED** [review]

**Pull request merged**